community/io/src/main/java/org/neo4j/io/pagecache/impl/muninn/StandalonePageCacheFactory.java

```diff
@@ -34,22 +34,27 @@
  */
 public final class StandalonePageCacheFactory
 {
-
     private StandalonePageCacheFactory()
     {
     }
 
     public static PageCache createPageCache( FileSystemAbstraction fileSystem )
     {
-        return createPageCache( fileSystem, PageCacheTracer.NULL );
+        return createPageCache( fileSystem, null, PageCacheTracer.NULL );
+    }
+
+    public static PageCache createPageCache( FileSystemAbstraction fileSystem, Integer pageSize )
+    {
+        return createPageCache( fileSystem, pageSize, PageCacheTracer.NULL );
     }
 
-    public static PageCache createPageCache( FileSystemAbstraction fileSystem, PageCacheTracer tracer )
+    public static PageCache createPageCache( FileSystemAbstraction fileSystem, Integer pageSize,
```

Comment:
Not too happy about this public method that does null logic on an Integer argument without any comment, I would like one of these to be fixed; for example add a comment, or add more constructors so that you can hide your null-magic in this class.